ouroboros-network/wireshark-plugin/ouroboros_network.lua

```diff
@@ -56,7 +63,8 @@ local txsubmission_msg_codes = {
 	[1] = "MsgReplyTxIds",
 	[2] = "MsgRequestTxs",
 	[3] = "MsgReplyTxs",
-	[4] = "MsgDone"
+	[4] = "MsgDone",
+        [6] = "MsgHello"
```

Comment:
Indentation.

Reply:
It seems this is caused by the endless war of `tabs` vs `spaces`.  I'd prefer to get rid of `tabs`, is that alright?